rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -571,7 +587,7 @@ def _create_label_data(
         needs_sentence_features = self._needs_sentence_features_for_labels()
         if not sentence_features and (not sequence_features or needs_sentence_features):
 
-            if sequence_features and not sentence_features and needs_sentence_features:
+            if not sequence_features and needs_sentence_features:
```

Comment:
```suggestion
            if sequence_features and needs_sentence_features:
```

Reply:
At least the warning suggests that this should be the condition 😅

Reply:
another ouch 🤦‍♀️  wanted to remove "not sentence_features" here....